packages/wdio-selenium-standalone-service/src/launcher.js

```diff
@@ -36,11 +37,18 @@ export default class SeleniumStandaloneLauncher {
          * update capability connection options to connect
          * to standalone server
          */
-        (
-            Array.isArray(this.capabilities)
-                ? this.capabilities
-                : Object.values(this.capabilities)
-        ).forEach((cap) => Object.assign(cap, DEFAULT_CONNECTION, { ...cap }))
+        let capabilities
+        if (Array.isArray(this.capabilities)) {
+            capabilities = this.capabilities
+        } else {
+            capabilities = Object.values(this.capabilities)
+        }
```

Comment:
This seems less elegant than `? :` because I used to need an `else if` in later condition. I can change back to `? :` if you prefer